src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,116 @@
+
+.card {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`

Reply:
i got. now have toubles to fix. thanks.